lte/cloud/go/services/nprobe/obsidian/handlers/handlers.go

```diff
@@ -0,0 +1,313 @@
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handlers
+
+import (
+	"math/rand"
+	"net/http"
+	"time"
+
+	"magma/lte/cloud/go/lte"
+	"magma/lte/cloud/go/serdes"
+	"magma/lte/cloud/go/services/lte/obsidian/handlers"
+	"magma/lte/cloud/go/services/nprobe/obsidian/models"
+
+	"magma/orc8r/cloud/go/obsidian"
+	"magma/orc8r/cloud/go/services/configurator"
+	"magma/orc8r/cloud/go/storage"
+	merrors "magma/orc8r/lib/go/errors"
+
+	strfmt "github.com/go-openapi/strfmt"
+	"github.com/labstack/echo"
+	"github.com/pkg/errors"
+)
+
+const (
+	NetworkProbeTasks                        = "network_probe_tasks"
+	ManageNetworkNetworkProbeTasksPath       = handlers.ManageNetworkPath + obsidian.UrlSep + NetworkProbeTasks
+	ManageNetworkNetworkProbeTaskDetailsPath = ManageNetworkNetworkProbeTasksPath + obsidian.UrlSep + ":task_id"
+
+	NetworkProbeDestinations                        = "network_probe_destinations"
+	ManageNetworkNetworkProbeDestinationsPath       = handlers.ManageNetworkPath + obsidian.UrlSep + NetworkProbeDestinations
+	ManageNetworkNetworkProbeDestinationDetailsPath = ManageNetworkNetworkProbeDestinationsPath + obsidian.UrlSep + ":destination_id"
+)
+
+func GetHandlers() []obsidian.Handler {
+	ret := []obsidian.Handler{
+		{Path: ManageNetworkNetworkProbeTasksPath, Methods: obsidian.GET, HandlerFunc: listNetworkProbeTasks},
+		{Path: ManageNetworkNetworkProbeTasksPath, Methods: obsidian.POST, HandlerFunc: createNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.GET, HandlerFunc: getNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteNetworkProbeTask},
+
+		{Path: ManageNetworkNetworkProbeDestinationsPath, Methods: obsidian.GET, HandlerFunc: listNetworkProbeDestinations},
+		{Path: ManageNetworkNetworkProbeDestinationsPath, Methods: obsidian.POST, HandlerFunc: createNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.GET, HandlerFunc: getNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteNetworkProbeDestination},
+	}
+	return ret
+}
+
+func getParamValues(c echo.Context, paramNames []string) ([]string, *echo.HTTPError) {
```

Comment:
I'd just get rid of this function. It's adding more complexity than needed. Just use `obsidian.GetParamValue(s)` directly at the call site.

Reply:
fixed.